src/main/java/com/mewna/catnip/CatnipOptions.java

```diff
@@ -199,6 +199,11 @@
      * catnip is not chunking members. Basically, this avoids a ton of logspam.
      */
     private boolean logUncachedPresenceWhenNotChunking = true;
+    /**
+     * If true, enables guild subscriptions flag in IDENTIFY payload which allows
```

Comment:
Should update the docs here to make it more specific how much is disabled, as it's a lot more than just typing/presence events according to the relevant PR to the API docs.